### PR TITLE
Trusted Intermediary: FHIR ServiceRequest -> OML Order and Order Observation

### PR DESCRIPTION
**Pull request by halprin**
This PR adds the conversion of the FHIR `ServiceRequest` resource to an HL7v2 `ORC` and `OBR`.

Test Steps:

```
wget https://raw.githubusercontent.com/CDCgov/trusted-intermediary/1c853caf380981e6c07b1725883616a34adeabf1/examples/fhir/lab_order.json
./gradlew primeCLI --args="fhirdata -i ./lab_order.json --output-format HL7 -s metadata/hl7_mapping/STLTs/CA/TrustedIntermediaryLabOrder.yml"
```

Example output

```
MSH|^~\&|CDC Trusted Intermediary^https://reportstream.cdc.gov/||||20230320100037-0500||OML^O21^OML_O21|969bcbb3-cd34-49be-ac4f-e1b8479b8219||2.5.1|||||USA|UNICODE UTF-8
PID|1||MRN7465737865^^^^MR||Solo^Jaina^^^^^L||20170515|F||^Asian^HL70005
NK1|1|Organa^Leia^^^^^L|N^^http://terminology.hl7.org/CodeSystem/v2-0131
ORC|NW||||IP||||20230320100037-0500
OBR||||54089-8^Newborn Screening Panel^LN|||||||P
```

## Changes

- Removes the `patient-visit` from `OML_O21-base.yml`.  We don't plan at this time to have an `Encounter` FHIR resource in our `Bundle`.
- Added the new `service-request-order.yml` and `service-request-order-observation.yml`.  These create an `ORC` and `OBR` segment.  I noticed that ReportStream already has `prime-router/metadata/hl7_mapping/ORU_R01/base/order.yml` (creates an `ORC`) and `prime-router/metadata/hl7_mapping/ORU_R01/base/observation-request.yml` (creates an `OBR`), but those are all centered around a `DiagnosticReport` FHIR resource, not a `ServiceRequest` FHIR resource.  You can see the official conversion from a `ServiceRequest` [to an `ORC`](https://docs.google.com/spreadsheets/d/11VBF5TIRJTfRwkRIK1XH-k40dZqCd3g3a0sBY-zJU4Q/edit#gid=0) and [to an `OBR`](https://docs.google.com/spreadsheets/d/1SW1j6qBHTbXJu4W56GCGMtO03ln3Z8ohyG9wY-AOX90/edit#gid=0).  I did not do an exhaustive conversion here, but only enough to support Trusted Intermediary for now.  We'd add more later as we find out more requirements.
- Added a TI-specific conversion into `TrustedIntermediaryLabOrder.yml` because the `OBR-11` conversion is open ended when the `ServiceRequest`'s `intent` is `order`.  Because TI knows that these orders we're generating are before the specimen arrives at the public health lab before the order, we hardcode `OBR-11` to `P` ([Pending specimen; Order sent prior to delivery](https://hl7-definition.caristix.com/v2/HL7v2.5.1/Tables/0065)).

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [x] Added tests?

### Process
- [x] Are there licensing issues with any new dependencies introduced?
- [x] Includes a summary of what a code reviewer should test/verify?
- [x] Updated the release notes?
- [x] Database changes are submitted as a separate PR?
- [x] DevOps team has been notified if PR requires ops support?

## Linked Issues

- Goes towards https://github.com/CDCgov/trusted-intermediary/issues/79

## To Be Done

_Nothing._

## Specific Security-related subjects a reviewer should pay specific attention to

- Does this PR introduce new endpoints?  _No._
- Does this PR include changes in authentication and/or authorization of existing endpoints?  _No._
- Does this change introduce new dependencies that need vetting?  _No._
- Does this change require changes to our infrastructure?  _No._
- Does logging contain sensitive data?  _No._
- Does this PR include or remove any sensitive information itself?  _No._

### CODE REVIEW

**Comment by arnejduranovic** [comment]
Have you run a sample FHIR file through and verified to output HL7? There should be steps in the description on how to get that file and test.

**Comment by halprin** [comment]
> Have you run a sample FHIR file through and verified the output HL7? There should be steps in the description on how to get that file and test.

I do.  [Here](https://raw.githubusercontent.com/CDCgov/trusted-intermediary/1c853caf380981e6c07b1725883616a34adeabf1/examples/fhir/lab_order.json).  Let me update the instructions above.

**Review by arnejduranovic — APPROVED** [review]

File: prime-router/metadata/hl7_mapping/OML_O21/OML_O21-base.yml [hunk 1]
```diff
@@ -32,11 +32,18 @@ elements:
       hl7SegmentGroup: '/PATIENT'
     schema: ../common/patient-contact
 
-  - name: patient-visit
```
[the remark this answers is not in the document]
Comment (halprin) [reply prime-router/metadata/hl7_mapping/OML_O21/OML_O21-base.yml:35]: I'm dumb.  I thought `Encounter` wasn't used with an `OML` conversion.  Relooking at the spec, though, [it is there](https://docs.google.com/spreadsheets/d/1cSYLIXVfIjl7ib2DEGxD96LD9-YVJNQtKul5IwzJtts/edit#gid=0&range=K20).  I'll put this back in.

[the remark this answers is not in the document]
Comment (halprin) [reply prime-router/metadata/hl7_mapping/OML_O21/OML_O21-base.yml]: Fixed.

**Pull request merged**